src/config.rs

```diff
@@ -265,6 +265,21 @@ impl Connection {
         }
     }
 
+    pub fn database_url_with_name(&self) -> std::string::String {
+        let name = self.name.as_ref().map_or(String::new(), |n| n.to_string());
+        let database_url = self.database_url().map_or(String::new(), |d| d.to_string());
```

Comment:
Could you change the return type to `anyhow::Result<String>`? That's because your implementation throws away errors.

```suggestion
    pub fn database_url_with_name(&self) -> anyhow::Result<String> {
        let database_url = self.database_url()?;
```

Reply:
Okay

Reply:
I fixed in this commit. Please check it.
https://github.com/TaKO8Ki/gobang/pull/129/commits/143350edcc35444432dee2eec749c5b5f5ceb824